src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -154,10 +157,13 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 			return;
 		}
 
+		StaticLoggerBinder.getSingleton().setMavenLog(getLog());
+		Logger logger = StaticLoggerBinder.getSingleton().getLoggerFactory().getLogger(getClass().getName());
```

Comment:
\[Checkstyle\] WARNING: 'method def' child has incorrect indentation level 16, expected level should be 4\.